vizro-core/src/vizro/models/_action/_action.py

```diff
@@ -35,6 +36,17 @@ class Action(VizroBaseModel):
         regex="^[a-zA-Z0-9_]+[.][a-zA-Z_]+$",
     )
 
+    # TODO: Abstract away once pattern becomes clear regarding function/import checks
```

Comment:
This comment is very vague and I think should be clearer. Imagine you're a junior developer who wasn't part of the conversation you and I had yesterday - what is actually undesirable with the code as it exists and what might possible solutions be?

Problem: generic `Action` model shouldn't depend on details of particular actions like export_data.
Possible solutions:  make a generic mapping of action functions to validation functions or the imports they require, and make the code here look up the appropriate validation using the function as key.